src/sentry/search/events/builder/base.py

```diff
@@ -1309,7 +1309,7 @@ def default_filter_converter(
         if search_filter.operator in ("=", "!=") and search_filter.value.value == "":
             if is_tag or is_attr or is_context or name in self.config.non_nullable_keys:
                 return Condition(lhs, Op(search_filter.operator), value)
-            else:
+            elif isinstance(lhs, Column):
```

Comment:
This should be column since it won't really work for Functions anyways

Reply:
What happens if this is an alias like a field alias or alised field?